libvirt.go

```diff
@@ -85,11 +84,38 @@ func (l *Libvirt) Capabilities() ([]byte, error) {
 // Connect establishes communication with the libvirt server.
 // The underlying libvirt socket connection must be previously established.
 func (l *Libvirt) Connect() error {
-	return l.connect()
+	payload := struct {
+		Padding [3]byte
+		Name    string
+		Flags   uint32
+	}{
+		Padding: [3]byte{0x1, 0x0, 0x0},
+		Name:    "qemu:///system",
```

Comment:
probably not the best time to address this, but: https://github.com/digitalocean/go-libvirt/issues/51

Reply:
Oh, yeah, that'd be a good one to address. I'll put together another PR for that in the next little while.